fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -3114,6 +3114,9 @@ public static int metaServiceRpcRetryTimes() {
     @ConfField(mutable = true, description = {"存算分离模式下calculate delete bitmap task 超时时间，默认15s"})
     public static int calculate_delete_bitmap_task_timeout_seconds = 15;
 
+    @ConfField(mutable = true, description = {"存算分离模式下事务导入calculate delete bitmap task 超时时间，默认300s"})
+    public static int calculate_delete_bitmap_task_timeout_seconds_for_transaction_load = 300;
```

Comment:
What about using calculate_delete_bitmap_task_timeout_seconds and adjust calculate_delete_bitmap_task_timeout_seconds to a larger number.